### PR TITLE
Upgrading Library

### PR DESCRIPTION
**Pull request by rscafi**
- [x] Update to PHP 7
- [x] Bump coverage
- [x] Code Style
- [x] Improve error handling
- [x] Update README
- [x] Add LICENSE and CODE_OF_CONDUCT

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 php:
-  - 5.4
-  - 5.5
+  - 7.0
+  - 7.1
 
 before_script:
   - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev
+  - php composer.phar install
```
Comment (gmsantos) [inline .travis.yml]: travis already include composer

File: composer.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "leroy-merlin-br/laravel-axado-api",
-    "description": "A consumer for axado API.",
-    "keywords": ["axado","laravel","shipping", "brazil", "laravel","illuminate"],
+    "description": "A consumer for Axado API.",
+    "keywords": ["axado","laravel","shipping", "brazil", "laravel","cep","quotation","frete","brasil"],
```
Comment (gmsantos) [inline composer.json]: why "brasil" and "brazil" ?

File: composer.json [hunk 3]
```diff
@@ -19,15 +19,21 @@
...
     },
+    "require": {
+        "php": ">=7.0",
+        "ext-curl": "*"
+    },
     "require-dev": {
-        "phpunit/phpunit": "~4.2",
-        "mockery/mockery": "dev-master",
+        "mockery/mockery": "^0.9",
+        "phpunit/phpunit": "^5.7",
```
Comment (gmsantos) [inline composer.json]: Any reason to not use phpunit 6 ?

File: tests/TestCase.php [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
-    }
+use Mockery as m;
+use PHPUnit\Framework\TestCase as BaseTestCase;
 
+class TestCase extends BaseTestCase
+{
+    /**
+     * {@inheritdoc}
+     */
     public function tearDown()
```
Comment (gmsantos) [inline tests/TestCase.php]: `protected function tearDown()` We dont need to a wide to public here.

File: composer.json (hunk 2 above)
Comment (rscafi) [reply composer.json]: *whispers: blackhat seo*

Comment (gmsantos) [reply composer.json]: Remove spaces, or add in all?
`
"keywords": ["axado","laravel","shipping","brazil","cep","quotation","frete","brasil"],
`

**Comment by rscafi** [comment]
Go home @coveralls, you are drunk

File: src/Quotation.php [hunk 5]
```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Axado;
+
+class Quotation
+{
+    /**
+     * Fields with be parsed after the response is returned.
```
Comment (vitorbari) [inline src/Quotation.php]: to be parsed?

**Pull request merged**